README.md

```diff
@@ -118,10 +89,14 @@ psql -d foia -c "CREATE USER foia WITH PASSWORD '<<PASSWORD>>';"
 django-admin.py syncdb
 ```
 
-* Finally, launch the server locally:
+* Update the `DATABASE_URL` in `.env` with your Postgres connection string.
+
+### Running the server
```

Comment:
Hmm, while I see your point, I also think it's important to verify as early as possible that the harder work (dependencies and project setup) are done. Could we merge this as-is and update it in a future PR?